apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.html.ts

```diff
@@ -40,59 +87,216 @@ export function InstructorAIGenerateDrafts({
+                    style="resize: none;"
+                  ></textarea>
+                  <div class="form-text form-muted">
+                    <em>
+                      Example: A toy car is pushed off a table with height h at speed v0. Assume
+                      acceleration due to gravity as 9.81 m/s^2. H is a number with 1 decimal digit
+                      selected at random between 1 and 2 meters. V0 is a an integer between 1 and 4
+                      m/s. How long does it take for the car to reach the ground?
+                    </em>
+                  </div>
+                </div>
+
+                <div class="js-hidden-inputs-container ${hasDrafts ? 'd-none' : ''}">
+                  <div class="form-group">
+                    <label for="user-prompt-llm-user-input">
+                      How should students input their solution? What choices or input boxes are they
+                      given?
+                    </label>
+                    <textarea
+                      name="prompt_user_input"
+                      id="user-prompt-llm-user-input"
+                      class="form-control js-textarea-autosize"
+                      style="resize: none;"
+                    ></textarea>
+                    <div class="form-text form-muted">
+                      <em>
+                        Example: students should enter the solution using a decimal number. The
+                        answer should be in seconds.
+                      </em>
+                    </div>
+                  </div>
+
+                  <div class="form-group">
+                    <label for="user-prompt-llm-grading">
+                      How is the correct answer determined?
+                    </label>
+                    <textarea
+                      name="prompt_grading"
+                      id="user-prompt-llm-grading"
+                      class="form-control js-textarea-autosize"
+                      style="resize: none;"
+                    ></textarea>
+                    <div class="form-text form-muted">
+                      <em>
+                        Example: the answer is computed as sqrt(2 * h / g) where g = 9.81 m/s^2
+                      </em>
+                    </div>
+                  </div>
+
+                  ${
+                    // We think this will mostly be useful in local dev or for
+                    // global admins who will want to iterate rapidly and don't
+                    // want to retype a whole prompt each time. For actual users,
+                    // we think this will mostly be confusing if we show it.
+                    resLocals.is_administrator
+                      ? html`
+                          <hr />
+
+                          <div class="mb-3">
+                            <label for="user-prompt-example" class="form-label">
+                              Or choose an example prompt:
```

Comment:
The example prompts don't work for me in either Safari or Chrome. The select is there and I can choose an example prompt, but it doesn't fill anything into the textboxes above.

Reply:
Fixed in c17bf8a.